### ISSUE TITLE
ChatGPT through webapp does not work

### ISSUE THREAD

**Issue by daffi7** [chathub-dev/chathub#555]
**Describe the bug**
ChatGPT does work through WebApp for me. That is a shame, because I would like to use its browsing mode. First I got the message which you get when you send more than one message at a time. But there was no other response being generated for me (not in any browser on my computer). 
Now I get the error: "Our systems have detected unusual activity from your system. Please try again later."
It just doesn't work. And it's funny it stopped working today as today was the day when I purchased my premium license :)
I had a proxy active so that Bard works for me (I live in the EU), even with it turned off it still doesn't work.
API access works, but I want the browsing feature.

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome

**Comment by wong2** [comment]
Hi, can you restart the browser and try again?

**Comment by jwno2** [comment]
thanks...but it seems not make sense that I need to pay if I have an account

**Comment by wong2** [comment]
@daffi7 I've found the cause of this issue, ChatGPT has added an additional challenge for GPT-4 API calls. I'm investigating ways to solve it.

**Comment by daffi7** [comment]
I have restarted everything, including the computer, disabled all chatgpt extensions (except ChatHub) and just now I can access GPT-4 browsing through the open AI web app chat.openai.com/chat, but when I tried it through ChatHub immediately after, it does not work. That is a disappointment as the browsing mode is my most favorite one. Without it, ChatHub Premium allowing to query 4 bots simultaneously is much less useful.
